app/ts/simulation/services/EthereumClientService.ts

```diff
@@ -210,51 +187,22 @@ export class EthereumClientService {
 		] } as const
 		const unvalidatedResult = await this.requestHandler.jsonRpcRequest(call)
 		/*
-		console.log('executionSpec383MultiCall')
+		console.log('ethSimulateV1')
```

Comment:
These look like debug codes that never fires anyway, let's rid if no longer used

Reply:
its code that I need to add there everytime there's issues with eth_simulateV1, so I rather keep it in.